frontend/src/routes/git-settings.tsx

```diff
@@ -45,9 +53,26 @@ function GitSettingsScreen() {
   const [bitbucketHostInputHasValue, setBitbucketHostInputHasValue] =
     React.useState(false);
 
-  const existingGithubHost = settings?.PROVIDER_TOKENS_SET.github;
-  const existingGitlabHost = settings?.PROVIDER_TOKENS_SET.gitlab;
-  const existingBitbucketHost = settings?.PROVIDER_TOKENS_SET.bitbucket;
+  const existingGithubHost = settings?.PROVIDER_TOKENS_SET.github?.host ?? null;
```

Comment:
this seems like a breaking change to the settings object--am I reading it wrong? I think this would break existing users